psi4/driver/procrouting/proc.py

```diff
@@ -3045,12 +3093,16 @@ def run_detci(name, **kwargs):
 
     ciwfn = core.detci(ref_wfn)
 
+    # Shove variables into global space
+    for k, v in ciwfn.variables().items():
+        core.set_variable(k, v)
```

Comment:
Is it possible to throw locals back into globals one level up, this seems like it could add a lot of code overall and make procreating even more complex.